### DISCUSSION TITLE
Update the "Resources Library" page on the wiki

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#2754] [General]
Please add the new scriptlets and examples of how to use them to [Resources Library](https://github.com/gorhill/uBlock/wiki/Resources-Library).

**Comment by MasterKia** [comment]
Added:
https://github.com/uBlockOrigin/uBlock-issues/wiki/Resources-Library/_compare/e7c90b103def88bb190700957d887e361e7b52af...e79da8583ae18439a5b11a95c82fc2379b0f7c93
